app/src/main/java/com/techcourse/dao/UserDao.java

```diff
@@ -1,121 +1,56 @@
 package com.techcourse.dao;
 
+import java.util.List;
+
 import com.techcourse.domain.User;
+
 import nextstep.jdbc.JdbcTemplate;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import javax.sql.DataSource;
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.util.List;
+import nextstep.jdbc.RowMapper;
 
 public class UserDao {
 
-    private static final Logger log = LoggerFactory.getLogger(UserDao.class);
+    private static final RowMapper<User> ROW_MAPPER = getMapper();
```

Comment:
```suggestion
    private static final RowMapper<User> ROW_MAPPER = (rs, rowNum) -> new User(rs.getLong(1),
            rs.getString(2),
            rs.getString(3),
            rs.getString(4));
```

위와 같이 작성할 수 있는데, 메서드로 분리하신 이유가 있을까요? 메서드로 선언하는 이유가 궁금합니다! 판다만의 스타일이신가요? 아니면 다른 장점이 있을까요? 궁금해서 여쭤봅니다!